pkg/oc/cli/set/volume.go

```diff
@@ -667,7 +667,7 @@ func (o *VolumeOptions) setVolumeMount(spec *kapi.PodSpec, info *resource.Info)
 
 	for _, c := range containers {
 		for _, m := range c.VolumeMounts {
-			if path.Clean(m.MountPath) == path.Clean(opts.MountPath) && m.Name != o.Name {
+			if filepath.Clean(m.MountPath) == filepath.Clean(opts.MountPath) && m.Name != o.Name {
```

Comment:
Hmm.... that's not quite correct, b/c in containers we always want to use forward slashes, that's why `path` was being used here. Have you got a chance to verify this on a windows machine?

Reply:
hm, i see.  I'll set up Win for testing, I started doing that but then this looked straight forward so was hoping to punt to QE - I'm unfamiliar w/ Win actually and don't have a development env set up in Win - only my kid's computer. I'll set it up, though.

Reply:
@soltysh I was able to test this in Windows :) while the path->path/filepath change results in no undesired prefix, it does end up with backwards rather than forward slash in the mountPath.  See here: https://bugzilla.redhat.com/show_bug.cgi?id=1807714#c10